### DISCUSSION TITLE
Tervezet - észrevételek

### DISCUSSION THREAD

**Discussion by tniLAB** [nav-gov-hu/eVAT#10] [Általános]
Üdv, Bátonyi Bálint vagyok, az alábbiakban az észrevételeim. (sok sikert kívánok a csapat előtt álló fejlesztésekhez)

"eÁfa M2M_tervezet.pdf"

**10.o. 1.ábra - 1.szint**
"Standard adókód lekérdezése"

Elnézést, ha triviális dolgokat feszegetek, de egy ideje nem töltök vissza OnlineSzámla felületről adatot és ezért inkább a logikai megközelítésről szól ez a megjegyzésem.

Az adókód adózói oldalon általában két alfanumerikus karakterből álló azonosító, amivel az összes bevallási adatot lehet kategorizálni és végső rendeltetési helyéhez rendelni. Multi cégeknél akár több EU-s országra vonatkozóan is. Például H1 a magyar telephelyhez rendelhető belföldi áfa-levonási joggal járó normálkulcsos termékbeszerzés. Ez a rendszer már több mint 10 éve megbízhatóan működik. Itt jön a képbe az, hogy nem láttam még a NAV saját adókód listáját (az onlineszamla.nav.gov.hu felület dokumentációiban sem), de a tervezet szövegéből úgy értettem, hogy létezik (ld még 12.o. 1.2 felsorolásában a standard adókódok/metaadatok letöltési lehetőségét és a 3.4, 4.1 bekezdéseket). Ez persze nem kell, hogy egyezzen az adózói adókód listákkal a tervezet szerint, de pont ez lenne a megfelelő lehetőség egy hatékonyan alkalmazható NAV adókód lista bevezetésére, melyet az adózók is (adózók alatt az adózók könyvelőit és adótanácsadóit is értem) használni szeretnének. Erre a listára való hivatkozást akár jövőbeni NAV/minisztériumi áfa állásfoglalások is tartalmazhatnák.

Az előző példánál maradva egy vázlatos felépítés:
1.karakter (irány): (I)ncoming, (O)outgoing
2.karakter (tranzakció): (S)ervices, (G)oods
3.karakter (másik fél lokáció): (D)omestic, (E)U, (N)onEU
4.karakter (kulcs): (N)ormal, (R)educed, (S)uperreduced, (E)xempt, (O)out of scope
5.karakter (levonhatóság): (D)eductable, (P)artial deductable, (N)ondeductable
6.karakter (tax point): (N)ormal, (I)ssuance date, (P)ayment date, (X) not classified

Vagyis a H1 kód besorolása: IGDNDN

A karakterlánc szerinti megközelítés azért jó, mert bővíthető (a régiek kompatibilitása biztosítható egy adott pozícióbeli X-el) és a mögöttes besorolási értékek 36-os (abc+10) listája is elégséges (ha nem, akkor új pozíció is nyitható ugyanarra a témára). A mögöttes magyarázat szintén bővíthető, verziókövethetően publikálható.

Úgy gondolom, egy ilyen lehetőséggel nagyon sok adózó élne, mert senkinek sem hiányzik a saját adókód kialakítási, ellenőrzési, (NAV) mappelési folyamatainak nyűgje.

**11.o. folyamat 5. pont**
"a könyvelő a programjában hoz döntést arról, hogy a bevallást az előzetes validációnak megfelelően beküldi-e"

Szerintem ezt nem szabadna döntésre bízni. Ha a validálni szánt bevallási pozíció helyes, akkor azt automatikusan beküldötté kellene minősíteni. Ha nem így történik, akkor azzal csak a trükközésnek és az önellenőrzések követhetetlenségének adunk teret (hiszen az önellenőrzést és az alapbevallást is egységesen kellene kezelni beküldési szempontból) és az ellenőrzések megkezdése szerinti bevallási állapot is kérdéseket vethet fel.

**11.o. folyamat 7. pont**
"Amennyiben a könyvelő az áfabevallás beadásának határidejét követő 10. nap éjféléig nem hozott döntést"

A többszázezres tételszámban (egy időszakra egy adózónak akár többszörösen is) tárolt állományok tárhelyének és beazonosíthatóságának, az egyéb bevallási csatornákkal történő szinkronizálásának valamint a hátralévő idő számításának a biztosítása is elkerülhető komplikáció az előző pont által.

**12.o. 1.1 bekezdés**
"kliens oldalon nincs semmilyen kötöttség a <Adattó lekérdezési szolgáltatások> használatukra"

Nagyon pozitív hozzáállás, hogy kötelezővé sem teszik, de nem is korlátozzák a használatukat.

**12.o. 1.1 b) pont**
"Warning információk elérhetővé tétele a vevő oldalon is"

Üzleti szempontból kényszeríti ki az adatszolgáltatási minőség javítását, jó kezdeményezés.

**14.o. 2.1 bekezdés**
"teljesítési időpont figyelembe vételével meghatározott időszakra történő lekérdezés biztosítása"

Szintén nagyon hasznos.

**15.o. 2.4 bekezdés**
"Ezek az adózói listák jellemzően adózónként és manuálisan kérdezhetők le, így azokat informatikai folyamatba nem lehet integrálni. Ezen adatbázisok közül néhánynak a gépi úton történő elérése szintén az adóhivatali célok között szerepel."

Az átmeneti időszakra, vagy akár API elérés helyett célrobotok publikálása is megoldás lehet, amik bár nem érik el egy API kapcsolat sebességét, de a manuális lekérdezésnél gyorsabbak, ütemezhetőbbek.

**16.o. 3.1 bekezdés**
"Az API response üzenet: amennyiben az eÁFA logikája meg tud határozni standard adókódot vagy adókódokat, akkor azok kerülnek visszaadásra az adott adókódhoz tartozó összeggel."

Érdemes lehet még az adókód mellé odatenni a bevallás kapcsolódó mezőinek sorszámát is (még akkor is, ha a webes áfaanalitikában ez az összerendelés részben megtalálható, de azt nem biztos, hogy minden esetben figyelembe veszik majd az adózók).

**16.o. 3.2 bekezdés**
"A két áfaanalitika független folyamat eredményeként jön létre és szerepe is eltérő egy adózó ügyviteli folyamatában"
Ez egy érdekes dilemma, de jelenleg még az egy darab helyes áfaanalitika elkészítése is jelentős részben leköti az adózói kapacitásokat.

"Amennyiben a felhasználó szabályokat definiált a felületen, azoknak megfelelően készül el a webes áfaanalitika"
Ezt is egy kicsit '2. analitikának" érzem (ideértve a 3.5 bekezdésben foglaltakat is), de az irány egyértelműen jó a gépi tanuláshoz szolgáltatott minták gyűjtése miatt. Talán a leginkább kapacitásigényes a szabályok dekódolása a webes felület környezetébe és az esetlegesen visszatöltött szabályok kielemzései alapján történő adózói változtatások menedzselése.

**17.o. 3.3 bekezdés**
"A warningok visszamutatása egyértelműen támogatja a jogkövető magatartást, esetleges önellenőrzések megfelelő adattartalommal történő beadását."

Ezeket a warning-okat már a validálás során megkaphatják az adózók, vagy csak a bevallási döntés meghozatalát követően? (a tervezet utolsó oldala szerint ez utóbbbi: "válaszüzenet, amely a beadás tényét tartalmazza. Ez a válaszüzenet tartalmazza az error vagy warning üzeneteket is.")

**19.o. 4.1 bekezdés**
"Mivel az adókódok teljesen kiváltják az adóbevallás űrlapot, ezért ez a tevékenység teljesen eltűnik, az adókódok standard adókóddal történő megfeleltetése jelentősen rövidebb időt igényel."

Már egy másik félmondat alapján is meg akartam kérdezni, hogy akkor a hagyományos űrlap formátum kivezetésre kerül-e vagy sem? Az (M-lapok nélküli) űrlap védelmében egy fontos funkciót megemlítenék: az xml nem képes arra, hogy a bevitt adatok valós időben változtassák a végső bevallási pozíciót, vagyis az adózó "nem látja" azt készítés közben. Persze le lehet erre is fejleszteni adózói xml-néző alkalmazásokat, de mivel azoknak az összes jelenlegi űrlap-alapú validálási szabályt követnie kell tudni (és az nem csekély komplexitás), nem biztos hogy ezt a funkciót az adózók széles rétege a továbbiakban is megengedheti-e magának.

**21.o. 4.2**
"Adóelszámolási információk:"

A 6-elemű felsorolásból a 2, 5 és 6 (adókulcs, adómentesség ok, levonási hányad) mind mehetne az adókód egyik karakterének. (ld az előbbi IGDNDN példámat)

**21.o. 4.2 utolsó bekezdés**

Az M-lapok hiánya elég erős hívószó lesz.

**22.o. 3.bekezdés**
<részleges időszaki validálás>

A leginkább előremutató döntésnek tartom, már én is régóta (2019-es cikkem is van erről) propagálom ezt a szemléletmódot a szinte valósidejű folyamatkorrekciókkal kapcsolatban.

**22.o. 4.bekezdés**
"Ugyanígy törlésre kerül az eÁFA M2M XML, amennyiben (...) nem adott be áfabevallást semmilyen formában sem és letelt az áfabevallás beadásának határidejét követő 10. nap."

Logikai typo: a semmilyen formában be nem adott bevallás nem létezik, tehát nem is törölhető.

**23.o. 5.2 2.bekezdés**
"Előfordulhat, hogy adózó néhány hónapja befogadott számlára alapoz levonási jogot, amely a kiajánlott adatokban nem is szerepel, azonban ennek a számlának nem szabad csupán ezen ok miatt warningot keletkeztetnie."

Én benne hagynám warning-ként, mert jelezhet rendszerhibát az adózónál és cash flow hátrányt is okoz.

**24.o. utolsó mondat**
"Az adózó döntése, hogy a warningok hatására önellenőrzést nyújt be, vagy a warningokat megvizsgálva arra az álláspontra jutott, hogy számára nem relevánsak."

Kiegészítve egy további esettel, amikor egy warning releváns, önellőrzés nem szükséges, viszont a warning-os adatokat előállító folyamatokat érdemes felülvizsgálni az adózónak.

**Comment by kdenes123** [comment]
Áfaanalitika - tételsor adatok

- Áfa főkönyvi számlaszám:
Ez nem minden esetben értelmezhető (pl. bevételi nyilvántartást vezetők), ill. ez egy könyvelési információ, amelynek nincs köze az áfa-bevalláshoz, valamint ez az áfa-kötelezettséget nem érintő okokból később változhat is.

- Főkönyvi feladás dátuma
Az áfaanalitika, a bevallási kötelezettség teljesítéséhez nem szükséges információ. Mit értünk főkönyvi feladáson? Kézi adatrögzítéskor nem kell ilyen adatot beküldeni?

- Levonási hányad...
Áfa-arányosítás esetén a számla könyvelésekor, az áfa-beállítások megadásakor még nem ismert a levonási hányad. Ez a megközelítés legfeljebb telefonszámlák, szgk. bérleti díjak esetén értelmezhető.

- Tranzakció leírás, megjegyzések
Az áfa-kötelezettség megállapításához nem szükséges információ. A könyvelésben általában nem rögzítik tételesen, szövegesen hogy milyen ügylet történik, így túl sok érdemi információ sem lesz már kinyerhető.

- Partner adatok
A partner adószáma egyértelműen azonosítja a vevőt/szállítót, ezért esetükben a név, cím felesleges.
Magánszemély vevők esetén az online adatszolgáltatás nem tartalmaz személyes információt. Adatvédelmi okokból az áfaanalitikában sem lehetne benne.
